src/main/java/org/opensearch/knn/index/SpaceType.java

```diff
@@ -36,6 +36,11 @@ public float scoreTranslation(float rawScore) {
         public VectorSimilarityFunction getVectorSimilarityFunction() {
             return VectorSimilarityFunction.EUCLIDEAN;
         }
+
+        @Override
+        public float scoreToDistanceTranslation(float score) {
+            return 1 / score - 1;
```

Comment:
Can we have a check to confirm score != 0?

Reply:
ack

Reply:
Added